mongodb/src/test/scala/com/stratio/crossdata/connector/mongodb/MongoImportTablesIT.scala

```diff
@@ -0,0 +1,129 @@
+  */
+package com.stratio.crossdata.connector.mongodb
+
+import com.stratio.datasource.mongodb.MongodbConfig
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+
+@RunWith(classOf[JUnitRunner])
+class MongoImportTablesIT extends MongoWithSharedContext {
+
+  it should "import all tables from MongoDB" in {
+    assumeEnvironmentIsUpAndRunning
+
+    xdContext.dropAllTables()
+
+    val importQuery =
+      s"""
+         |IMPORT TABLES
+         |USING $SourceProvider
+         |OPTIONS (
+         |host '$MongoHost:${MongoPort.toString}',
+         |${MongodbConfig.SamplingRatio} '0.1'
+         |)
+      """.stripMargin
+
+    sql(importQuery)
+    sql("SHOW TABLES").count() should be > (0l)
+
+  }
+
+  it should "import tables from a MongoDB database" in {
+    assumeEnvironmentIsUpAndRunning
+
+    xdContext.dropAllTables()
+
+    val importQuery =
+      s"""
+         |IMPORT TABLES
+         |USING $SourceProvider
+         |OPTIONS (
+         |host '$MongoHost:${MongoPort.toString}',
+         |${MongodbConfig.Database} '$Database',
+         |${MongodbConfig.SamplingRatio} '0.1'
+         |)
+      """.stripMargin
+
+    sql(importQuery)
+    sql("SHOW TABLES").count() should be > (0l)
+
+  }
+
+  it should "import unique table from a MongoDB collection" in {
+    assumeEnvironmentIsUpAndRunning
+
+    xdContext.dropAllTables()
+
+    val importQuery =
+      s"""
+         |IMPORT TABLES
+         |USING $SourceProvider
+         |OPTIONS (
+         |host '$MongoHost:${MongoPort.toString}',
+         |${MongodbConfig.Database} '$Database',
+         |${MongodbConfig.Collection} '$Collection',
+         |${MongodbConfig.SamplingRatio} '0.1'
+         |)
+      """.stripMargin
+
+    sql(importQuery)
+    sql("SHOW TABLES").count() should be (1l)
+  }
+
+  it should "import table from a collection with incorrect database" in {
+    assumeEnvironmentIsUpAndRunning
+
+    xdContext.dropAllTables()
+    val wrongCollection = "wrongCollection"
+    val importQuery =
+      s"""
+         |IMPORT TABLES
+         |USING $SourceProvider
+         |OPTIONS (
+         |host '$MongoHost:${MongoPort.toString}',
+         |${MongodbConfig.Database} '$Database',
+         |${MongodbConfig.Collection} '$wrongCollection',
+         |${MongodbConfig.SamplingRatio} '0.1'
+         |)
+      """.stripMargin
+
+    sql(importQuery)
+    sql("SHOW TABLES").count() should be (0l)
+  }
+
+  it should "import unique table from a MongoDB collection without indicate the database" in {
+    assumeEnvironmentIsUpAndRunning
+
+    xdContext.dropAllTables()
+
+    val importQuery =
+      s"""
+         |IMPORT TABLES
+         |USING $SourceProvider
+         |OPTIONS (
+         |host '$MongoHost:${MongoPort.toString}',
+         |${MongodbConfig.Collection} '$Collection',
+         |${MongodbConfig.SamplingRatio} '0.1'
+         |)
+      """.stripMargin
+
+    sql(importQuery)
+    sql("SHOW TABLES").count() should be > 0l
```

Comment:
could you add assertions about tables data instead the number of tables?  like confirms that table "A" is listed and table "B" aren't.